src/stratis_cli/_actions/_physical.py

```diff
@@ -27,8 +27,21 @@
 from ._data import ObjectManager
 from ._data import Pool
 from ._data import pools
+from ._data import MODev
+from ._data import devs
+from ._formatting import print_table
 
 
+def state_num_to_string(val):
+    states = {
+        0: "Missing",
+        1: "Bad",
+        2: "Spare",
+        3: "Not-in-use",
+        4: "In-use",
+    }
+    return states.get(val, "Unknown")
```

Comment:
Better to raise exception on unknown value.

Reply:
I disagree, because of the context this value is used. It's just informational. We could raise an exception, but then in the caller we're just going to catch it and convert back into an "UNEXPECTED" string to print, why not just do that in the function?